src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/VirtualSecureModeEnclaveProvider.cs

```diff
@@ -57,6 +58,8 @@ protected override string GetAttestationUrl(string attestationUrl)
         }
 
         // Makes a web request to the provided url and returns the response as a byte[]
+        [SuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
+                   Justification = "DataContractJsonSerializer used here deserialize only byte[] which is already kept.")]
```

Comment:
Use the local function refactoring to apply the suppression only onto the specific piece of code (in this case the two lines inside the using statement)